iamproperty/iamproperty.github.io
# Buttons

## Summary of Changes:
1. Documentation buttons and links
2. Visual regression

## Rationale
Document how buttons are used and modified.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/buttons. 

## Ticket(s)
DS-36

## Checklist

## Notes
-
